src/spyglass/spikesorting/merge.py

```diff
@@ -114,3 +112,15 @@ def get_firing_rate(cls, key, time, multiunit=False):
             ],
             axis=1,
         )
+
+
+@schema
+class UnitInclusion(SpyglassMixin, dj.Manual):
```

Comment:
Do we want to shelve this until after the current PR?

Reply:
We are still seeing how far we can get but I can remove if we don't end up using it right away.